speakers.md

```diff
@@ -0,0 +1,52 @@
+# Submitting A Talk
+
+## Who can submit talks to HH.js?
+We welcome and encourage _everyone_ to submit a talk!
+Whether you have been writing JavaScript for 20 years or few days, experienced or 1st time speaker, we want to hear what you are excited about.
+
+## How do I submit a talk?
+Visit our proposals GitHub repository and [create a new issue](https://github.com/hamburg-js/proposals/issues/new/choose) using the appropriate template. Make sure to pick up to 3 months that work for you!
+
+## What would be a good talk topic?
+We accept a variety of technical and non-technical talks and the audience has varying technical backgrounds and interests.
+
+Talks from accepted proposals are usually focused on, but not limited to:
+- Sharing the process and journey of how you created something and what you've learned.
+- Achieving empathy in the workplace.
+- A dive into a particular technical concept.
+
+Examples of topics that may not get accepted:
+- Why x framework is better than y framework.
+- Startup or company pitches.
+
+ We've found that the best talks tell a story, we encourage you to get creative!
+
+## When are speakers chosen?
+We usually select speakers two weeks before an event.
+
+_Current employees of hosts are not selected to speak on the months of their employer's sponsorship._
+
+## I have more questions about submitting talks, what should I do?
+Please feel free to ping us on twitter [@hhjs](https://twitter.com/hhjs) or join us in our [WebHH Slack](https://web-hh-slackin.herokuapp.com/) group [#hhjs](https://web-hh.slack.com/messages/C0KESM96C)
+
+# Speaking at HH.js
+We are excited to have you as speaker! Here is few things to keep in mind when you are making your talk. If you have any questions or want to discuss your talk with organizers, please feel free to ping us!
```

Comment:
```diff
- Here is few things to keep in mind when you are making your talk.
+ Here are a few things to keep in mind when giving your talk.
```

```diff
- …, please feel free to ping us!
+ …, please feel free to ping us via <…>!
```

Reply:
I've changed these sentences a bit, please take another look.

As for your second suggestion: What should we put there? The `#hhjs` slack channel? Or tell them to ping us via private messages on slack?

Reply:
I would be fine with Slack.

Reply:
I linked this with the other paragraph that contains all the links to twitter, slack, etc.